charts/prometheus-thanos/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.14.0"
```

Comment:
this should be 0.17.2. too.

Reply:
Sorry about that! 🤦 

I've sorted with last commit.  While we're here I've also added support for multiple querier replica labels.  It's feasible that people will want to add more than 2 of these flags, i.e. one with the previous `.Values.querier.additionalFlag` variable one more with `.Values.querier.additionalFlags` (since this is a map)